<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QGroupBox" name="tipoLomito">
      <property name="title">
       <string>Elija tipo de lomito:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="lomitoCarne">
         <property name="text">
          <string>Carne - $500</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="lomitoPollo">
         <property name="text">
          <string>Pollo - $450</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="lomitoCerdo">
         <property name="text">
          <string>Cerdo - $400</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="extras">
      <property name="title">
       <string>Elija extras:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="huevoFrito">
         <property name="text">
          <string>Huevo frito - $100</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="papasFritas">
         <property name="text">
          <string>Papas fritas - $200</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="gaseosa">
         <property name="text">
          <string>Gaseosa - $250</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="delivery">
         <property name="text">
          <string>Delivery - $100</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="precioTotal">
      <property name="text">
       <string>Precio total:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCalcular">
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
